addons/analytic/views/analytic_distribution_model_views.xml

```diff
@@ -21,14 +21,18 @@
             <form string="Analytic Distribution Model">
                 <sheet>
                     <group>
+                        <field name="analytic_distribution_stored_char" invisible="1"/>
+                        <span class="text-muted mt-4 mb-2">Simultaneous conditions to meet</span><br/>
                         <group>
                             <field name="partner_id"/>
                             <field name="partner_category_id"/>
-                            <field name="company_id" groups="base.group_multi_company"/>
                         </group>
                         <group>
-                            <field name="analytic_distribution_stored_char" invisible="1"/>
-                            <field name="analytic_distribution" string="Analytic Distribution" widget="analytic_distribution"
+                            <field name="company_id" groups="base.group_multi_company"/>
+                        </group>
+                        <span class="text-muted mt-4 mb-2">Analytic distribution to apply</span>
+                        <group colspan="2">
```

Comment:
Do we not want `<group string="Analytic distribution to apply"`?
Or does it have to be `muted`?

Reply:
This could work too, I will ask his opinion to benj once I have the final mockups :) Thanks!